pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1543,7 +1543,7 @@ func (og *operationGenerator) GenerateExpandVolumeFunc(
 				return volumetypes.NewOperationContext(detailedErr, detailedErr, migrated)
 			}
 			successMsg := fmt.Sprintf("ExpandVolume succeeded for volume %s", util.GetPersistentVolumeClaimQualifiedName(pvc))
-			og.recorder.Eventf(pvc, v1.EventTypeNormal, kevents.VolumeResizeSuccess, successMsg)
+			og.recorder.Eventf(pvc, nil, v1.EventTypeNormal, kevents.VolumeResizeSuccess, "", successMsg)
```

Comment:
Do we want to keep this as an empty string? I see below on line 1568 we provide the string "Expanding Volume" for a similar kubelet event type.